src/services/user/services/AdminUsers.js

```diff
@@ -1,107 +1,124 @@
 /* eslint-disable no-param-reassign */
 /* eslint-disable class-methods-use-this */
-const { BadRequest, Forbidden } = require('@feathersjs/errors');
+const { BadRequest, Forbidden, GeneralError } = require('@feathersjs/errors');
```

Comment:
Codacy found an issue: ['BadRequest' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)